package.json

```diff
@@ -285,7 +285,6 @@
     "stylelint": "6.9.0",
     "supertest": "2.0.0",
     "terminal-kit": "1.13.13",
-    "uglify-js": "2.7.0",
```

Comment:
Do we not need this anymore?

Reply:
Yes we do. The diff is tricky to see, but it's just moved from `devDependencies` to `dependencies`. This was breaking local Docker builds because it's directly invoked by `bin/minify.js`, which is part of the `build-client-if-prod` script, which is called after `npm install --prod` in the Dockerfile. As a dev dep, it was being omitted and the build was broken without actually failing.